content/streamlit-cloud/get-started/deploy-an-app/index.md

```diff
@@ -17,20 +17,40 @@ If you want to deploy your app on a different cloud service, check out the [Depl
 
 Streamlit Community Cloud launches apps directly from your GitHub repo, so your app code and dependencies need to be on GitHub before you try to deploy the app. See [App dependencies](/streamlit-community-cloud/get-started/deploy-an-app/app-dependencies) for more information.
 
-### Optionally, add a configuration file
+```
+your-repository/
+├── your_app.py
+└── requirements.txt
+```
+
+If you are including any custom [Configuration](/library/advanced-features/configuration) or [Themeing](/library/advanced-features/theming), make sure your config file is saved relative to the root of your repo. Within your repo, your config file should be `.streamlit/config.toml`.
 
-Streamlit allows you to optionally set configuration options via four different methods. Among other things, you can use custom configs to customize your app's theme, enable logging, or set the port on which your app runs. For more information, see [Configuration](/library/advanced-features/configuration) and [Theming](/library/advanced-features/theming). On Streamlit Community Cloud, however, you can only set configuration options via a configuration file in your GitHub repo.
+```
+your-repository/
+├── .streamlit/
+│   └── config.toml
+├── your_app.py
+└── requirements.txt
+```
 
-Specifically, you can add a configuration file to the root (top-level) directory of your repo: create a `.streamlit` folder, and then add a `config.toml` file to that folder. E.g., if your app is in a repo called `my-app`, you would add a file called `my-app/.streamlit/config.toml`. Say you want to set the theme of your app to "dark". You would add the following to your `.streamlit/config.toml` file:
+If your main app file is not located at the root of your repository, your folder structure might look like:
```

Comment:
Would leave out example 3, looks a bit confusing in my eyes.

Reply:
Done. I think I'll add a KB article to my tasks to show how the different combinations of things look (multipage, configuration, Python dependencies, apt-get dependencies, multiple apps in a repository, and apps not in root.